files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml

```
@@ -606,3 +669,181 @@ tools:
     rules:
       - if: input_size >= 30
         cores: 12
+  toolshed.g2.bx.psu.edu/repos/bgruening/xchem_transfs_scoring/xchem_transfs_scoring/.*:
+    scheduling:
+      require:
+        - docker
+  toolshed.g2.bx.psu.edu/repos/bgruening/openduck_run_smd/openduck_run_smd/.*:
+    env:
+      docker_set_user: 1000
+      docker_run_extra_arguments: '-e "OPENDUCK_GPU_PARAM=$OPENDUCK_GPU_PARAM" --gpus all'
+    scheduling:
+      require:
+        - docker
+  toolshed.g2.bx.psu.edu/repos/bgruening-util/stress_ng/stress_ng/.*:
+    scheduling:
+      require:
+        - singularity
+        - conda
+  toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant/.*:
+    scheduling:
+      require:
+        - singularity
+  toolshed.g2.bx.psu.edu/repos/iuc/lumpy_prep/lumpy_prep/.*:
+    scheduling:
+      require:
+        - singularity
+        - conda
+  # is there a way to avoid this
+  "*.pcgr.*":
```

drop? I did not find a long name.

---

It is this tool here, that we deploy locally https://github.com/bgruening/galaxytools/pull/1257  
@wm75 do we have plans for it?

---

If no long id neither exists nor is planned by @wm75, then the following should work.

```suggestion
  ^pcgr$:
```